src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -505,6 +505,22 @@ public void GetAsync_CredentialIsCredentialCacheUriRedirect_HttpStatusCodeOK()
             }
         }
 
+        [Theory, MemberData("PostServers")]
```

Comment:
This test is defined inside of a #region marked for Put tests.  It should be moved up to where the other Post tests are.